fuels-abigen-macro/tests/test_projects/empty-arguments/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "method_four_arguments"
+version = "0.1.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+license = "Apache-2.0"
+
+[dependencies]
+fuels-abigen-macro = "0.2"
+fuels-core = "0.2"
+fuels-rs = "0.2"
```

Comment:
Shouldn't this use a local dep instead of a pinned dep? You want this test to test the current SDK.

Reply:
True! Using `forc init` is a trap!